cmd/update-notifier.go

```diff
@@ -36,14 +37,15 @@ func colorizeUpdateMessage(updateString string, newerThan time.Duration) string
 	// Calculate length without color coding, due to ANSI color
 	// characters padded to actual string the final length is wrong
 	// than the original string length.
-	hTime := timeDurationToHumanizedDuration(newerThan)
-	line1Str := fmt.Sprintf(" Minio is %s old ", hTime.StringShort())
+	newerThanStr := humanize.Time(UTCNow().Add(newerThan))
```

Comment:
`currentBuildTime` and `currentReleaseTime` are in UTC.  `newerThan` is a difference of those two time.  In this function, we are printing how much newer `currentReleaseTime`.  Whether the duration is from `currentBuildTime` or `UTCNow()` makes no difference.